src/apm.coffee

```diff
@@ -48,6 +48,34 @@ module.exports =
           programFilesPath = path.join(process.env.ProgramFiles, 'Atom', 'resources', 'app.asar')
           callback(programFilesPath)
 
+  getResourcePathSync: (callback) ->
```

Comment:
I was thinking about that too, but this approach seemed to have less of a ripple effect.